mmdeploy/codebase/mmdet3d/deploy/voxel_detection_model.py

```diff
@@ -93,12 +93,15 @@ def forward(self,
                 'coors': coors
             }
             outputs = self.wrapper(input_dict)
-            result = VoxelDetectionModel.post_process(self.model_cfg,
```

Comment:
As `post_process` is not used now, should we remove it?

Reply:
ok, I will remove post_process.